index.html

```diff
@@ -44,7 +233,47 @@ <h1> I’m George
                 <li> <a href="#"><img src="img/Vectorch.png" alt="icon"></a> </li>
             </ul>
         </div>
+        <div class="action">
+            <div class="normal-btn">
+                <a href="#" class="btn-text">Get my resume</a>
+            </div>
+        </div>
+        <br>
+
+        <div class="language">
+            <P><b>Languages</b></P>
+            <p id="rotate"> > </p>
+        </div>
+        <br><br>
+        <div class="lang-list">
+            <ul class="a-items">
+                <li class="b-items"> <img src="img/js.png"> <a href="#" id="icon-text"> JavaScrip</a> </li>
+                <li class="b-items"> <img src="img/html.png"> <a href="#" id="icon-text"> HTML</a> </li>
+                <li class="b-items"> <img src="img/css.png"> <a href="#" id="icon-text"> CSS</a> </li>
+            </ul>
+        </div>
+        <div class="frame-8">
+            <ul class="frameworks">
+                <li><p class="frames">Frameworks</p></li>
+                <li><img src="img/Disabledforward.png" alt="forward" class="forward"></li>
+            </ul>
+                       
+            <div class="line"></div>
+        </div>
+
+        <div class="frame-8">
+            <ul class="frameworks">
+                <li><p class="frames">Skills</p></li>
+                <li><img src="img/Disabledforward.png" alt="forward" class="forward"></li>
+            </ul>
+                       
+            <div class="line"></div>
+        </div>
+        
     </section>
```

Comment:
- [ ] Good job so far 👏. From the markup guidelines, this section is expected to be implemented as a `nested list` . The technology categories (e.g: `Languages`, `Frameworks`, etc) should be in list `<ul>` where each is in `<li>` but also the technology(eg: `HTML`, `CSS`, etc) in each category should be list `<ul>` where each is in `<li>`. I know it may sound difficult to implement that is why there is [an article](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/) that will help you get the idea of how to implement it 😉